src/sources/audio_file/source.rs

```diff
@@ -5,7 +5,7 @@ use crate::backend::AudioBackend;
 use crate::prelude::{AudioFileError, AudioFileSettings, AudioSource};
 use crate::sources::audio_file;
 use bevy::asset::Asset;
-use bevy::prelude::TypePath;
+use bevy::prelude::*;
```

Comment:
I don't mind this kind of stuff but I know others might. Just wanted to make sure this change is intentional

Reply:
Oh yeah that's definitely unintentional (oops auto imports) 😅